### PR TITLE
Filter fields

### PR DESCRIPTION
**Pull request by shimastripe**
## WHY
- Filter fields in the response data.

## WHAT
- implement a `?q[field_name]` query.
- remove `?ids` query - This is almost the same query.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ Resource name is written in the plural form.
 
 |Parameter|Description|Default|Example|
 |---------|-----------|-------|-------|
-|`ids=`|Item IDs|(empty)|`1,2,5`|
+|`q[field_name]=`|A unique query parameter for each field for filtering|(empty)|`1,2,5`|
```
Comment (dtan4) [inline README.md]: Maybe it's more useful to suggest example with query key,

`q[id]=1,2,5`, `q[admin]=true&q[registered]=true`

**Comment by dtan4** [comment]
Please merge the latest master and resolve conflicts.

**Comment by shimastripe** [comment]
Merge branch `master`.
Please review again.

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**